ionic_user_interface/src/views/About.vue

```diff
@@ -4,19 +4,34 @@
       <div id="container">
         <strong>Hello Climbers! &#129495;</strong>
         <p class="lead">
-          Personally, when people make routes for me at the gym,
-          <br />
-          I tend to forget half of the moves due to my poor memory.
+          This is a free open source project between two friends who happen to code and climb.
         </p>
 
         <p class="lead">
-          One way is to draw them out on a handphone,
+          It aims to accomplish two things:
           <br />
-          but it is difficult to do so with fat fingers, and labeling
+        </p>
+        <p class="lead">
```

Comment:
For this section, can left-align and maybe put a border on it, also add some spacing between 1. and 2., and maybe use 1.<tab>XXX instead of 1.<space>XXX